contrib/make_locale

```diff
@@ -12,8 +13,8 @@ original_dir = os.getcwd()
 os.chdir(os.path.dirname(os.path.realpath(__file__)))
 os.chdir('..')
 
-code_directories = 'gui plugins lib ios/ElectronCash/electroncash_gui/ios_native'
-cmd = "find {} -type f -name '*.py' -o -name '*.kv'".format(code_directories)
+code_directories = ['android/app/src/main/python', 'gui', 'ios/ElectronCash', 'lib', 'plugins']
```

Comment:
The code directory for iOS needs to be what it was previously.  If you can, kindly set it back.

Explanation: This is because as you are working the `ios/ElectronCash` directory could potentially contain duplicate files copied over by `make_ios_project.sh` because of the way Briefcase works (they needed to be copied in there).

If you can, please set it back to what it was or I will have to remember to after this is merged.

Reply:
OK, done.